chart/templates/deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
                   operator: In
                   values:
                   - {{ template "rancher.fullname" . }}
-              topologyKey: kubernetes.io/hostname
+              topologyKey: {{ .Values.topologyKey }}
```

Comment:
What happens if it is omitted? Is that valid or should we default?

Reply:
We need to have a default value for topologyKey.
@JustInVTime Can you update your PR to add a default value as my suggested changes indicate?